language-support/qute/qute.tmLanguage.json

```diff
@@ -118,7 +124,7 @@
           "name": "keyword.control"
         },
         "3": {
-          "name": "keyword.control"
+          "name": "entity.name.tag.identifier"
```

Comment:
let's keep `keyword.control` to have the same color for hash and section name.

Reply:
This will mean `default` and `user` tags will have the same colour, is that okay?

Reply:
No default tag is keyword.control and user tag is entity.name.function.identifier

Reply:
but every `#` and `/` should be `keyword.control` correct?